druid-shell/src/platform/windows/window.rs

```diff
@@ -670,7 +676,7 @@ impl WndProc for MyWndProc {
             WM_ACTIVATE => {
                 if LOWORD(wparam as u32) as u32 != 0 {
                     unsafe {
-                        if !self.has_titlebar() {
+                        if !self.has_titlebar() && !self.is_transparent() {
```

Comment:
why should it necessarily be transparent to ignore drop shadows? This also depends on the application, if its 100% transparent shadows would look weird but if its only 10% transparent it would still be nice to have shadows.

Reply:
Yeah, I think it's just not possible to run `DwmExtendFrameIntoClientArea` twice. To get the transparency it's already executed above with negative margins.